guides/user/pipeline/triggers/index.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: single
+title:  "Overview"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+A pipeline trigger defines when to automatically run a pipeline. There are a
+wide variety of triggers available: Jenkins jobs, webhooks, CRON jobs, even
+other pipelines. Adding a trigger to your pipeline means that the pipeline will
+run each time the triggering event occurs. For more information about how to
+configure specific types of triggers, see the rest of the pipeline triggers
+documentation.
```

Comment:
I think, end this with a colon, then list out the sub-sections of the triggers doc.

"...see the rest of the triggers documentation:
* Triggering on Pub/Sub messages
* Triggering on Webhooks"

(and of course link to those articles)

And maybe put in a ToDo: add to the list when other articles are added, like "Triggering a pipeline from another pipeline"